doc/nrf/libraries/others/date_time.rst

```diff
@@ -37,6 +45,10 @@ Configuration
 
    Configure this option to control the frequency with which the library fetches the time information.
 
+:kconfig:`CONFIG_DATE_TIME_AUTO_UPDATE`
+
+   Configure this option to determine whether date-time update is triggered automatically when device has connected to LTE.
+
```

Comment:
Maybe an entry to the changelog is needed.

Reply:
Done